packages/@aws-cdk/aws-events/lib/rule.ts

```diff
@@ -37,30 +37,34 @@ export interface RuleProps {
 
   /**
    * The schedule or rate (frequency) that determines when EventBridge
-   * runs the rule. For more information, see Schedule Expression Syntax for
+   * runs the rule.
+   *
+   * You must specify this property, the `eventPattern` property, or both.
+   *
+   * Note that rates cannot be defined in fractions of minutes.
```

Comment:
Doesn't it make more sense to put this on `Schedule.rate()` ?

Reply:
It does, putting it on `Schedule.rate()` instead.